server/api/forms/db/index.js

```diff
@@ -0,0 +1,17 @@
+const { Pool } = require('pg')
+
+const pool = new Pool(
+  {
+    user: 'postgres',
+    host: 'localhost',
+    database: 'test',
+    password: 'password',
```

Comment:
Consider instead of hard coding these, import them as environmental variables. See `/config/index.js` for examples of this. Define development host as `process.env.POSTGRES_HOST_DEV` and production host as `process.env.POSTGRES_HOST_PRODUCTION`, for example.